src/test/docker/docker-compose.yml

```diff
@@ -11,3 +11,7 @@ services:
         ports:
             - "7000:3100"
         command: -config.file=/etc/loki/local-config.yaml
+    workspace:
```

Comment:
I do not mind the change but you should also update the README.md to reflect this.  
Please add the information needed to run the phpunit suite through the docker container you added (CLI instructions will suffice).
Thank you

Reply:
@rrajkomar I just updated the readme. Please check if this is ok for you.

Reply:
Thanks